### PR TITLE
[SYCL][UR][L0] Don't execute loop for printing if UR_DEBUG not enabled

### PR DESCRIPTION
**Pull request by jandres742**

### CODE REVIEW

**Comment by jandres742** [comment]
failures unrelated:

this is only failing with OpenCL
```
********************
Failed Tests (1):
  SYCL :: ESIMD/accessor_local.cpp
```

failures with HIP and CUDA shouldn't happen, as here we are touching only L0 files.

File: sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_event.cpp [hunk 1]
```diff
@@ -16,13 +16,18 @@
...
 
 void printZeEventList(const _ur_ze_event_list_t &UrZeEventList) {
-  urPrint("  NumEventsInWaitList %d:", UrZeEventList.Length);
+  if (UrL0Debug & UR_L0_DEBUG_BASIC) {
+    urPrint("  NumEventsInWaitList %d:", UrZeEventList.Length);
 
-  for (uint32_t I = 0; I < UrZeEventList.Length; I++) {
-    urPrint(" %#llx", ur_cast<std::uintptr_t>(UrZeEventList.ZeEventList[I]));
-  }
+    if (UrL0Debug & UR_L0_DEBUG_BASIC) {
```
Comment (againull) [inline sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_event.cpp:22]: It looks like this "if" is redundant, could you please remove it if so.

Comment (jandres742) [reply sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_event.cpp:22]: thanks @againull :) yes, missed that. removed

**Review by againull — APPROVED** [review]

**Comment by jandres742** [comment]
@intel/llvm-gatekeepers : please merge when possible.

**Pull request merged**